### PR TITLE
adds strip metadata recipe

### PR DESCRIPTION
**Pull request by ablwr**

### CODE REVIEW

**Comment by privatezero** [comment]
👍🚀

**Comment by kfrn** [comment]
Is it for stripping embedded descriptive/XMP-based metadata?

**Comment by ablwr** [comment]
Yeah... it also overwrites things like muxing/writing app (since it is making a copy).

**Comment by kfrn** [comment]
Very cool!

File: index.html [hunk 1]
```diff
@@ -2156,6 +2156,29 @@
...
+        <div class="well">
+          <h3>Strips metadata from video file</h3>
+          <p><code>ffmpeg -i <i>input_file</i> -map_metadata -1 -vcodec copy -acodec copy test-nometadata.mov</code></p>
+          <dl>
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt>-i <i>input_file</i></dt><dd>path, name and extension of the input file</dd>
+            <dt>-map_metadata -1</dt><dd>sets metadata copying to -1, which copies nothing</dd>
+            <dt>-vcodec copy</dt><dd>copies video track</dd>
+            <dt>-acodec copy</dt><dd>copies audio track</dd>
+            <dt>-o <i>output_file</i></dt><dd>Makes copy of original file and names output file</dd>
```
Comment (retokromer) [inline index.html]: Why is there a `-o` parameter?

**Review by retokromer — CHANGES_REQUESTED** [review]
nitpicking…

File: index.html [hunk 2]
```diff
@@ -2156,6 +2156,29 @@
...
   <!-- ends Generate Video Fingerprint -->
 
+  <!-- Strip metadata -->
+  <span data-toggle="modal" data-target="#strip_metadata"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Strip metadata">Strip metadata</button></span>
+  <div id="strip_metadata" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Strips metadata from video file</h3>
+          <p><code>ffmpeg -i <i>input_file</i> -map_metadata -1 -vcodec copy -acodec copy test-nometadata.mov</code></p>
```
Comment (retokromer) [inline index.html]: Either we add a note explaining that `-vcodec` and `-acodec` is the same than `-c:v` and `-c:a`, or we unify.

File: index.html (hunk 1 above)
Comment (ablwr) [reply index.html]: Oh because I have been working with other microservices wayyyy too much...!

**Comment by ablwr** [comment]
Was I asleep when I made this PR? Haha, please re-review. 😅

File: index.html [hunk 3]
```diff
@@ -2163,14 +2163,14 @@
       <div class="modal-content">
         <div class="well">
           <h3>Strips metadata from video file</h3>
-          <p><code>ffmpeg -i <i>input_file</i> -map_metadata -1 -vcodec copy -acodec copy test-nometadata.mov</code></p>
+          <p><code>ffmpeg -i <i>input_file</i> -map_metadata -1 -c:v copy -c:a copy <i>input_file</i></code></p>
```
Comment (retokromer) [inline index.html]: The second `input_file` should read `output_file`.

Comment (ablwr) [reply index.html]: Ah, turns out I wasn't sleepy but trying to work during a meeting... ;) One moment!

Comment (retokromer) [reply index.html]: Take two moments ;-)

**Review by retokromer — APPROVED** [review]

**Comment by retokromer** [comment]
Thank you!

**Comment by ablwr** [comment]
Haha thanks for tolerating my lack of 💯 attention!

**Pull request merged**